integration-tests/src/main/scala/za/co/absa/spline/test/fixture/SparkFixture.scala

```diff
@@ -22,21 +22,27 @@ import org.scalatest.{Assertion, AsyncTestSuite, BeforeAndAfterEach}
 import za.co.absa.commons.io.TempDirectory
 
 import java.io.File
+import java.net.URI
 import java.sql.DriverManager
 import scala.concurrent.Future
 import scala.util.Try
 
 trait SparkFixture {
   this: AsyncTestSuite =>
 
-  val warehouseDir: String = TempDirectory("SparkFixture", "UnitTest", pathOnly = true).deleteOnExit().path.toString.stripSuffix("/")
+  val baseDir: TempDirectory = TempDirectory("SparkFixture", "UnitTest", pathOnly = true).deleteOnExit()
+  val warehouseDir: String = baseDir.asString.stripSuffix("/")
+  val warehouseUri: URI = baseDir.toURI
+  val metastoreDir: String = TempDirectory("Metastore", "debug", pathOnly = true).deleteOnExit().asString
+  val metastoreConnectURL = s"jdbc:derby:$metastoreDir/metastore_db;create=true"
 
   protected val sessionBuilder: SparkSession.Builder = {
     SparkSession.builder
       .master("local")
       .config("spark.driver.host","localhost")
       .config("spark.sql.warehouse.dir", warehouseDir)
       .config("spark.ui.enabled", "false")
+      .config("javax.jdo.option.ConnectionURL", metastoreConnectURL)
```

Comment:
What is this config needed for?

Reply:
It is used to define custom paths to metastore_db. It solves an issue found on windows where default location was still hold by process when trying to apply clean logic.